### DISCUSSION TITLE
Deprecate bitwise operators

### DISCUSSION THREAD

**Discussion by Denneisk** [wiremod/wire#2834] [Suggestions]
~~Replace them with `bit<Op>(a, b?)` functions such as `bitAnd(a, b)`, `bitNot(a)`.~~
They already have replacements.

**Comment by vurvdev** [comment]
They already exist fyi

https://github.com/wiremod/wire/blob/master/lua/entities/gmod_wire_expression2/core/bitwise.lua

**Reply by Denneisk** [reply]
Wow
